unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BackpackEditorHUDV2/VRMExport/VRMDetailsComponentView.cs

```diff
@@ -0,0 +1,136 @@
+        [SerializeField] private VRMDetailItemComponentView vrmDetailPrefab;
+        [SerializeField] private Transform wearablesContainer;
+
+        private Pool wearableDetailPool;
+        private readonly List<PoolableObject> pooledItems = new ();
+        private readonly List<string> equippedItemsURNs = new();
+
+        public event Action OnBackButtonPressed;
+        public event Action OnVRMExportButtonPressed;
+        public event Action<VRMItemModel, UnequipWearableSource> OnWearableUnequipped;
+        public event Action<VRMItemModel, EquipWearableSource> OnWearableEquipped;
+
+        public override void Awake()
+        {
+            base.Awake();
+            vrmExportButton.onClick.RemoveAllListeners();
+            vrmExportButton.onClick.AddListener(OnVRMExportButtonClick);
+
+            backButton.onClick.RemoveAllListeners();
+            backButton.onClick.AddListener(() => OnBackButtonPressed?.Invoke());
+
+            wearableDetailPool = GetItemDetailPool();
+        }
+
+        public override void Dispose()
+        {
+            base.Dispose();
+            var pool = GetItemDetailPool();
+
+            foreach (PoolableObject pooledItem in pooledItems)
+                pool.Release(pooledItem);
+        }
+
+        private Pool GetItemDetailPool()
+        {
+            var entryPool = PoolManager.i.GetPool(VRM_EXPORT_DETAILS_POOL_ID);
+            if (entryPool != null) return entryPool;
+
+            entryPool = PoolManager.i.AddPool(
+                VRM_EXPORT_DETAILS_POOL_ID,
+                Instantiate(vrmDetailPrefab).gameObject,
+                maxPrewarmCount: 5,
+                isPersistent: true);
+
+            entryPool.ForcePrewarm();
+
+            return entryPool;
+        }
+
+        private void OnVRMExportButtonClick()
+        {
+            OnVRMExportButtonPressed?.Invoke();
+        }
+
+        private void EnableVRMExport(bool enable)
+        {
+            vrmExportButton.SetInteractable(enable);
+            exportWarningBar.SetActive(!enable);
+        }
+
+        public override void RefreshControl() { }
+
+        public override void OnDisable()
+        {
+            base.OnDisable();
+
+            foreach (PoolableObject pooledItem in pooledItems)
+                pooledItem.Release();
+        }
+
+        public void FillVRMBlockingWearablesList(List<NFTDataDTO> itemsToDisplay)
+        {
+            EnableVRMExport(false);
+            equippedItemsURNs.Clear();
+
+            foreach (var itemData in itemsToDisplay)
+            {
+                equippedItemsURNs.Add(itemData.urn);
+                var detailItem = wearableDetailPool.Get();
+                detailItem.gameObject.transform.SetParent(wearablesContainer, false);
+                pooledItems.Add(detailItem);
+
+                var detailComponentView = detailItem.gameObject.GetComponent<VRMDetailItemComponentView>();
+
+                if (detailComponentView)
```

Comment:
Is this case possible? If thats the case you will miss to show the item

Reply:
It shouldn't be possible, but I should also enforce it with a `[RequiresComponent]` tag in the prefab 👌 but regardless, good catch! That `if` was a leftover during testing this algorithm 💪